templates/VALIDATION.md

```diff
@@ -51,12 +51,12 @@ If a rule is meaningless, for example if it has no arguments (see below), its be
 ### JSONPaths
 
 For every jsonpath mentioned in this document, unless specified otherwise, the root is the objects: element of the template.
-Unless otherwise specified, the jsonpath must be enclosed in double brackets - this is to disambiguate plain strings from jsonpaths.
-Example:
+Unless otherwise specified, the value to be used as jsonpath must be prefixed with the "jsonpath::" literal.
```

Comment:
This confused me a bit when I looked at the updated example above. I think we should make it clear that this only applies to the value field. All other fields like valid or path do not need this prefix.